nanomq/conf_api.c

```diff
@@ -460,6 +468,45 @@ set_reload_config(cJSON *json, conf *config)
 	if (rv == 0) {
 		update_var(config->backoff, backoff);
 	}
+	getBoolValue(json, item, "enable_mqtt_stream", enable_mqtt_stream, rv);
+	if (rv == 0) {
+		bool enable_mqtt_stream_before = config->parquet.enable;
+		// Update first. Is error happened in follow codes. revert.
+		update_var(config->parquet.enable, enable_mqtt_stream);
+
+		if (enable_mqtt_stream_before == true && enable_mqtt_stream == false) {
+			if (hook_sock == NULL) {
+				hook_sock = nng_alloc(sizeof(nng_socket));
+				if (0 != nng_push0_open(hook_sock)) {
```

Comment:
how about put this sock to conf?
Now it create a new socket each time calling HTTP